packages/git/node-tests/indexer-remote-test.js

```diff
@@ -201,7 +201,7 @@ describe('git/indexer cloning', function() {
     await start();
     await indexer.update();
 
-    let contents = await searcher.get(env.session, 'master', 'events', 'event-1');
+    let contents = await searcher.get(env.session, 'local-hub', 'events', 'event-1');
     let jsonapi = toResource(contents);
```

Comment:
@robustdj for the `searchers.get()` we no longer take a branch parameter, instead of that, we now just pass the hard coded string `local-hub`, which will be useful in the future to specifying getting a model from a foreign hub instance. for now we only support your local hub instance.